### PR TITLE
Changed the log window on the web UI to scroll through logs rather than use pagination

### PR DESCRIPTION
**Pull request by daichij**

### CODE REVIEW

File: server/resources/js/app/widb.js [hunk 1]
```diff
@@ -21,8 +21,15 @@ class WirelessDebug {
     /** @private @const {?MetricGrapher} */
     this.metricGrapher = null;
 
+    this.tableConfig = {
```
Comment (sumnerevans) [inline server/resources/js/app/widb.js]: JSDoc.

File: server/resources/js/app/widb.js [hunk 2]
```diff
@@ -21,8 +21,15 @@ class WirelessDebug {
...
+      'paging': false,
+      'lengthMenu': [-1],
+      'scrollY': '75vh',
+      'scrollCollapse': true,
+    };
+
     /** @private @const {!DataTable} */
-    this.dataTable = $('#log-table').DataTable();
+    this.dataTable;;
```
Comment (sumnerevans) [inline server/resources/js/app/widb.js]: Remove one of the `;`s

File: server/resources/js/app/widb.js [hunk 3]
```diff
@@ -90,7 +97,9 @@ class WirelessDebug {
     if (messageData.messageType === 'logData') {
       this.data_table.destroy();
       this.logTable_.append(messageData.logEntries);
-      this.data_table = $('#log-table').DataTable();
+      this.data_table = $('#log-table').DataTable(this.tableConfig);
+      let scrollBody = $('.dataTables_scrollBody');
```
Comment (sumnerevans) [inline server/resources/js/app/widb.js]: Add a comment about what you are doing to this line.

File: server/resources/js/app/widb.js (hunk 1 above)
Comment (daichij) [inline server/resources/js/app/widb.js]: How does one JSDoc?

File: server/resources/js/app/widb.js [hunk 4]
```diff
@@ -21,8 +21,16 @@ class WirelessDebug {
...
+    this.tableConfig_ = {
+      'paging': false,
+      'lengthMenu': [-1],
+      'scrollY': '75vh',
+      'scrollCollapse': true,
+    };
+
     /** @private @const {!DataTable} */
-    this.dataTable = $('#log-table').DataTable();
+    this.dataTable;
```
Comment (cnavrides) [inline server/resources/js/app/widb.js]: Wait... How does this work? Shouldn't this be titled this.data_table? Or is it a different from the data_table on line number 63?

Comment (cnavrides) [inline server/resources/js/app/widb.js]: And if it is private const (like the comment suggests), shouldn't it end with _?

File: server/resources/js/app/widb.js [hunk 5]
```diff
@@ -88,9 +96,11 @@ class WirelessDebug {
     let messageData = JSON.parse(message.data);
 
     if (messageData.messageType === 'logData') {
-      this.data_table.destroy();
-      this.logTable_.append(messageData.logEntries);
-      this.data_table = $('#log-table').DataTable();
+      // If we get more log data, append the log data to the table and scroll to
+      // the bottom of the table.
+      this.data_table.row.add($(messageData.logEntries)).draw();
+      let scrollBody = $('.dataTables_scrollBody');
```
Comment (cnavrides) [inline server/resources/js/app/widb.js]: Can you make it consistent what you are using in terms of camelCase vs _ seperated. Style guide says to use camelCase http://google.github.io/styleguide/javascriptguide.xml#Naming

**Comment by sumnerevans** [comment]
@daichij you have a typo in the title of this PR: "user" -> "use"

**Pull request merged**